src/templates/macros.java

```diff
@@ -22,27 +22,25 @@
     UniffiHelpers.uniffiRustCall(
     {%- endmatch %} _status ->
     UniffiLib.INSTANCE.{{ func.ffi_func().name() }}(
+        {# TODO(murph): this `it` doesn't exist #}
```

Comment:
How do you want to resolve this? It seems like we should have a test that covers it.

Reply:
It does actually exist. This macro is called from the `to_ffi_call` macro just above, where if `func.takes_self()` is true, `it` gets manually set.